pkg/fission-cli/logdb/influxdb.go

```diff
@@ -158,18 +159,24 @@ func (influx InfluxDB) GetLogs(ctx context.Context, filter LogFilter, output *by
 	return nil
 }
 
-func (influx InfluxDB) query(query influxdbClient.Query) (*influxdbClient.Response, error) {
-	queryURL, err := url.Parse(influx.endpoint)
-	if err != nil {
-		return nil, err
+func (influx InfluxDB) query(ctx context.Context, query influxdbClient.Query) (*influxdbClient.Response, error) {
+
+	dbType := INFLUXDB
+	// retrieve db auth config from the env
+	url := os.Getenv(fmt.Sprintf("%s_URL", dbType))
+
+	if url == "" {
+		// set up default database url
```

Comment:
This code should be part of NewInfluxDb only I guess.
From lines 164-174.
```
	dbType := INFLUXDB
	// retrieve db auth config from the env
	url := os.Getenv(fmt.Sprintf("%s_URL", dbType))

	if url == "" {
		// set up default database URL 
				url = influx.endpoint
	}
	username := os.Getenv(fmt.Sprintf("%s_USERNAME", dbType))
	password := os.Getenv(fmt.Sprintf("%s_PASSWORD", dbType))
```	
We should not do port forwarding if influxdb URL is set.

The query should only create a new request and fire it.